src/app/helptext/sharing/iscsi/iscsi.ts

```diff
@@ -317,5 +315,44 @@ export const helptext_sharing_iscsi = {
     title: T('Updated'),
     text: T('Fibre Channel '),
     msg: T(' update successful')
+  },
+
+  // wizard
+  step1_label: T("Create or Choose Block Device"),
+
+  name_placeholder: T("Name"),
+  name_tooltip: T("Keep the name short. Using a name longer than 63 characters can prevent accessing the block device."),
+
+  disk_placeholder: T("Device"),
+  disk_tooltip: T("Select the unformatted disk, controller, zvol, or zvol snapshot. Select\
+ <i>Create New</i> for options to create a new zvol."),
+
+  dataset_placeholder: T("Pool/Dataset"),
+  dataset_tooltip: T("Browse to an existing pool or dataset to store the new zvol."),
+
+  volsize_placeholder: T("Size"),
+  volsize_tooltip: T("Specify the size of the new zvol."),
+
+  volblocksize_placeholder: T("Block Size"),
+  volblocksize_tooltip: T("Only override the default if the initiator requires a different block size."),
+
+  usefor_placeholder: T("What are you using this for"),
```

Comment:
```suggestion
  usefor_placeholder: T("What are you using this for?"),
```
I don't particularly like this placeholder, but I can't come up with a solid alternative in the 10-20 seconds I have available to think about this. "Platform for Share", "Sharing Platform", eh

Reply:
Thanks @Mrt134, changed it to 'Sharing Platform'.